crashreports/internal/unix/CrashReportContext.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <string>
+
+class CrashReportContext
+{
+    static constexpr int MaxBufferLength{ 2048 };
```

Comment:
@gera-gas and in fact, it would be nicer to use size_t not int, when the intention is a constant that describes an array bound.  Because size_t is supposed to be the type sufficient to describe the size of any contiguous array in memory.

So either
static constexpr size_t MaxBufferLength{ 2048 };

or

enum : size_t { ...

(I hope we all know that notation to specify the underlying type of an enum.  You need it to properly forward-declare an enum type without the enumerators.)